File: src/Models/db/finance/Payout.ts
import {PaymentStatus} from "../../../Enums/PaymentStatus";

export interface Payout {
    id: number;
    payout_batch_id: string;
    user_id: number;
    amount_ct: number;
    currency_code: string;
    stripe_transfer_id: string | null;
    stripe_payout_id: string | null;
    reversed_amount_ct: number;
    status: PaymentStatus;
    created_at: Date;
    updated_at: Date;
}

File: kotlin/com/example/lydaandroid/model/Payout.kt
package com.example.lydaandroid.model

import java.util.Date

data class Payout(
    val id: Int,
    val payout_batch_id: String?,
    val user_id: Int,
    val amount_ct: Int,
    val status: PaymentStatus,
    val created_at: Date,
    val updated_at: Date
)
